packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -772,7 +807,13 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
                'list-person': true
              };
              return html`
-               <li class="${classMap(listPersonClasses)}" @click="${e => this.onPersonClick(person)}">
+               <li
+                role="option"
+                aria-label=" suggested contact ${person.displayName}"
```

Comment:
I'm wondering if localization would be an issue. Could we at least use the LocalizationHelper here?

Reply:
Definitely an issue! I'm putting in an update.